src/ng/directive/ngOptions.js

```diff
@@ -303,6 +303,7 @@ var ngOptionsDirective = ['$compile', '$parse', function($compile, $parse) {
         values = values || [];
 
         Object.keys(values).forEach(function getWatchable(key) {
+          if (key.charAt(0) === '$') return;
```

Comment:
Do we really want to ignore `$` (and not just `$$`) properties ?
`$$` are (by convention) private, but `$` are (by convention again) Angular-specific, but not private.